homeassistant/components/kulersky/light.py

```diff
@@ -156,8 +156,8 @@ async def async_update(self):
         self._available = True
         brightness = max(rgbw)
         if not brightness:
-            rgbw_normalized = [0, 0, 0, 0]
+            self._attr_rgbw_color = (0, 0, 0, 0)
         else:
-            rgbw_normalized = [round(x * 255 / brightness) for x in rgbw]
+            rgbw_normalized = tuple(round(x * 255 / brightness) for x in rgbw)
+            self._attr_rgbw_color = rgbw_normalized  # type:ignore[assignment]
```

Comment:
Why do we ignore type?

Reply:
`rgbw_normalized` is a `tuple` of `int`, but without a length (`tuple[int, ...]`).
So the type:ignore is needed because it expects a fixed length tuple `tuple[int, int, int, int]`

Reply:
Can we cast? Adding a type ignore is not an improvement.

Reply:
I'm happy to replace it with a cast, but if mypy improves the understanding of the comprehension, then the type ignore will be flagged as unecessary and will be dropped. Whereas the cast is more likely to stay there indefinitely.

Reply:
Isn't the problem that the library isn't typed? If we cast `rgbw` it could work here?

Reply:
I'm still trying to figure out a better way to do this.
The issue is that `rgbw` is already correctly typed as `tuple[int, int, int, int]`. But there is a routine to "normalize" it.
Would something like this be better?
```python
            self._attr_rgbw_color = (
                _normalize(rgbw[0], brightness),
                _normalize(rgbw[1], brightness),
                _normalize(rgbw[2], brightness),
                _normalize(rgbw[3], brightness),
            )
```

Reply:
How can `rgbw` be correctly typed? The library doesn't have type annotations.